### DISCUSSION TITLE
Redirect with fragment

### DISCUSSION THREAD

**Discussion by shankhadevpadam** [livewire/livewire#8836] [3. Feature ideas]
The current version of Livewire doesn't support withFragment on redirection. Can it be possible to support withFragment in the next version of Livewire?

**Comment by sgtcoder** [comment]
You can use withFragment

Here is a function you can put in your functions js file.
```
window.scroll_to_hash = function () {
    if (window.location.hash) {
        var target = $(window.location.hash);

        $("html,body").animate(
            {
                scrollTop: target.offset().top,
            },
            750
        );
    }
};
```

Put this on your layout that runs on every page that calls that function
```
    <script type="module">
        $(function() {
            scroll_to_hash();
        });
    </script>
```
